pytorch_lightning/loops/processors/iterator_batch_processor.py

```diff
@@ -114,14 +108,33 @@ def get_active_optimizers(self, batch_idx: Optional[int] = None) -> List[Tuple[i
         """
         return list(enumerate(self.trainer.optimizers))
 
+    def _build_kwargs(self, dataloader_iter: Iterator, batch_idx: int) -> Dict[str, Any]:
```

Comment:
shouldn't this go to the bottom to the protected section?

Reply:
Done ! However, this entire component will be dropped in next PR, which is why I didn't bother :)